analytics/index.yaml

```diff
@@ -0,0 +1,727 @@
+          type: string
+          format: hexadecimal
+          minimum: 6
+          maximum: 6
+          example: ff5c6d
+        - name: idgoal
+          in: formData
+          #TODO: Unify Ecommerce naming scheme. Rest of parameters refer to Ecommerce order, not cart.
+          description: |
+            <p>Goal ID.</p>
+            <p>Signifies that a goal has been reached and tracks it's conversion.</p>
+            <p>Value "0" is reserved for Ecommerce (cart or order) tracking.</p>
+          #TODO: Confirm format of a goal ID.
+          type: integer
+          example: 15
+        - name: revenue
+          in: formData
+          #TODO: Confirm revenue format. Is it in cents?
+          description: |
+            <p>Revenue value of accieved goal.</p>
+            <p>Currency of the value doesn't matter, but only one should be used by the application (e.g. USD).</p>
+            <p>It's recommended to use lowest nominal available in selected currency (e.g. 4.99 USD tracked as 499 US
+            cents).</p>
+          type: integer
+          example: 499
+        - name: ec_id
+          in: formData
+          description: Ecommerce order ID.
+          #TODO: Confirm type is correct.
+          type: integer
+          example: 3352
+        - name: ec_items
+          in: formData
+          #TODO: Confirm format in example is valid.
+          #TODO: Confirm that all attributes of item are required.
+          #TODO: Add limits to the item attributes (e.g. string max length).
+          description: |
+            <p>Ecommerce order item list.</p>
+            <p>Each item on the list must contain "sku" - string (stock keeping unit), "name" - string,
+            "category" - string, "price" - integer and "quantity" - integer.</p>
+            <p>Currency of the price value doesn't matter, but only one should be used by the application (e.g. USD).</p>
+            <p>It's recommended to use lowest nominal available in selected currency (e.g. 4.99 USD tracked as 499 US
+            cents).</p>
+          type: string
+          #TODO: See JSON comment.
+          format: JSON
+          example:
+            - sku: "craft-311"
+              name: "Unicorn Iron on Patch"
+              category: "Crafts & Sewing"
+              price: 499
+              quantity: 3
+            - sku: "craft-312"
+              name: "Unicorn Lamp"
+              category: "Crafts & Sewing"
+              price: 1325
+              quantity: 1
+        - name: ec_st
+          in: formData
+          description: |
+            <p>Ecommerce order sub total (order cost without shipping).</p>
+            <p>Currency of the value doesn't matter, but only one should be used by the application (e.g. USD).</p>
+            <p>It's recommended to use lowest nominal available in selected currency (e.g. 4.99 USD tracked as 499 US
+            cents).</p>
+          type: integer
```

Comment:
@sebastianpiskorski I assumed `integer` values for monetary parameters, but JS suggests `float`. Can you confirm that type is used/recommended?